config/webpack.dev.js

```diff
@@ -146,6 +146,7 @@ module.exports = function (options) {
       new DefinePlugin({
         'ENV': JSON.stringify(METADATA.ENV),
         'HMR': METADATA.HMR,
+		'IS_PROD': false,
```

Comment:
This has to be JSON.stringify(false). Please read the documentation for DefinePlugin.